website/versioned_docs/version-v2.0_alpha/developers-references/smart-contracts/Gatekeepers.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Gatekeepers Smart Contract
+description: Gatekeepers smart contract
+sidebar_label: Gatekeepers
+sidebar_position: 6
+---
+
+MACI requires a signup gatekeeper to ensure that only designed users register. It is up to MACI's deployer how they wish to allow sign-ups, therefore they can implement their own GateKeeper. The repository comes with different options:
+
+- `FreeForAllSignUpGatekeeper` - This allows anyone to signup on MACI.
+- `SignUpTokenGatekeeper` - This makes use of a ERC721 token to gatekeep the signup function.
+- `EASGatekeeper` - This allows gatekeeping signups to only users who have a specific EAS attestation.
+- `HatsGatekeeper` - This allows gatekeeping signups to only users who have a specific Hat.
```

Comment:
Do we need to add gitcoin gatekeeper keeper here?

Reply:
yup good catch will add